helm/cluster-openstack/templates/kubeadm_control_plane.yaml

```diff
@@ -13,7 +13,12 @@ spec:
           - "api.{{ include "resource.default.name" $ }}.{{ .Values.baseDomain }}"
         extraArgs:
           cloud-provider: external
+          enable-admission-plugins: "NamespaceLifecycle,LimitRanger,ServiceAccount,ResourceQuota,PersistentVolumeClaimResize,DefaultStorageClass,Priority,DefaultTolerationSeconds,MutatingAdmissionWebhook,ValidatingAdmissionWebhook"
```

Comment:
can we make this configurable? it would be great for SA as many times we need to test new one, or add it for a customer PoC